optuna/integration/chainermn.py

```diff
@@ -147,6 +148,11 @@ def optimize(
                     pass
                 except catch:
                     pass
+                finally:
+                    # The following line mitigates memory problems due to container virtualization.
```

Comment:
I think that the problem being addressed in #325 is not specific to container virtualization environments (at least in theory). Therefore, I prefer an expression that covers a broader range such as "memory problems that can be occurred in some environments".

Reply:
Thank you for your suggestion. I mostly agree with you, but I feel that 'some environments' is ambiguous.
I think adding an example like 'some environments (e.g., services that use computing containers such as CircleCI)'  is useful for developers.
What do you think?

Reply:
Sounds good.

Reply:
Thank you. I'll fix the comments.

Reply:
I've updated the comments at the commit https://github.com/pfnet/optuna/pull/377/commits/0b8e509c5f24d5ad2120a5592af8940126ba9a9c.